terraform/service_apiserver.tf

```diff
@@ -44,18 +44,6 @@ resource "google_secret_manager_secret_iam_member" "apiserver-db" {
   member    = "serviceAccount:${google_service_account.apiserver.email}"
 }
 
-resource "google_secret_manager_secret_iam_member" "apiserver-twilio" {
```

Comment:
I believe we still need this. Otherwise the server can't actually access the secret.

Reply:
as discussed - this is now per-realm, so we'll use envelope encryption and store creds in DB